sriov_cni/pkg/utils/testing.go

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
license header?

Reply:
these are upstream sriov-cni code. We won't change this here.